### ISSUE TITLE
NTR: Cardiac Imaging: valvular vegetation (Cardiac & vascular)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7790]
New term request Cardiac Imaging: valvular vegetation (brumme_220419183559): Echocardiogram; cardiac ultrasound; intracardiac foreign object; intracardiac mass  UMLS: 1399207

**Comment by pnrobinson** [comment]
@MickeySegal
how can we differentiate this from 
[Libman-Sacks lesions HP:0011714](https://hpo.jax.org/app/browse/term/HP:0011714),
which is perhaps a bit more specific?

**Comment by MickeySegal** [comment]
We use this finding for various forms of endocarditis (infective (L or R heart) or nonbacterial) and for invasive candidiasis.  It would be good to have a general term as a parent of Libman-Sacks.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000326
- label: Cardiac valve vegetations
- parents: [Abnormal heart valve morphology (HP:0001654)](https://hpo.jax.org/app/browse/term/HP:0001654)
- pmid: PMID:17678415
- def: Endocarditis infection occurs along the edges of the heart valves. Damage to the cardiac endothelium followed by platelet and fibrin deposition results in a sterile lesion known as nonbacterial thrombotic endocarditis (NBTE) or vegetation. Microbes entering the bloodstream via routine or invasive procedures adhere to the vegetation and are engulfed in an outer meshwork of fibrin and platelets. Subsequent rapid microbial multiplication in a protected area of impaired host defense leads to vegetation growth and continuous bacteremia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7790
- synonyms:
